url-updater/src/main/java/com/devonfw/tools/ide/url/updater/gcviewer/GcViewerUrlUpdater.java

```diff
@@ -30,4 +35,14 @@ protected String getGithubRepository() {
 
     return "GCViewer";
   }
+
+  @Override
+  protected void collectVersionsFromJson(GithubTags jsonItem, Collection<String> versions) {
+
+    for (GithubTag item : jsonItem) {
+      String version = item.getRef().replace("refs/tags/", "");
+      if(version.matches("\\d+\\.\\d+(\\.\\d+)?"))
```

Comment:
How should the versions be filtered in general? Should we add a condition here in this function or use "mapVersion" in general?

Reply:
Why do you have to duplicate this at all?
https://github.com/devonfw/ide/blob/169da0cc26df54921ee82d466de8a47d52a9e1a9/url-updater/src/main/java/com/devonfw/tools/ide/url/updater/GithubUrlUpdater.java#L29

Reply:
There are some versions that are crawled, but they dont match the version format as 2.5.6

I could also use the `mapVersion` function, but I thought that we can already filter the versions before we save them. Or am I miss something here?

Reply:
I use the mapVersion function